soh/.gitignore

```
@@ -18,6 +18,7 @@ notes/
 baserom/
 docs/doxygen/
 *.elf
+*.arm64
```

```suggestion
*.arm64
*.x86_64
```

Do you need to add *.x86_64 for Intel builds?